coordinator/src/routes.rs

```diff
@@ -542,3 +553,55 @@ pub async fn collaborative_revert_confirm(
     })?;
     Ok(Json(serialize_hex(&raw_tx)))
 }
+
+// TODO(holzeis): There is no reason the backup and restore api has to run on the coordinator. On
+// the contrary it would be much more reasonable to have the backup and restore api run separately.
+pub async fn backup(
+    Path(node_id): Path<String>,
+    State(state): State<Arc<AppState>>,
+    backup: Json<Backup>,
+) -> Result<(), AppError> {
+    let node_id = PublicKey::from_str(&node_id)
+        .map_err(|e| AppError::BadRequest(format!("Failed to backup {}. {e:#}", backup.key)))?;
+
+    state
+        .user_backup
+        .backup(node_id, backup.0)
+        .await
+        .map_err(|e| AppError::InternalServerError(e.to_string()))
+}
+
+// TODO(holzeis): There is no reason the backup and restore api has to run on the coordinator. On
+// the contrary it would be much more reasonable to have the backup and restore api run separately.
+pub async fn delete_backup(
+    Path(node_id): Path<String>,
+    State(state): State<Arc<AppState>>,
+    backup: Json<DeleteBackup>,
+) -> Result<(), AppError> {
+    let node_id = PublicKey::from_str(&node_id).map_err(|e| {
+        AppError::BadRequest(format!("Failed to delete backup {}. {e:#}", backup.key))
```

Comment:
Nit: 
```suggestion
AppError::BadRequest(format!("Invalid node id provided {}. {e:#}", backup.key))
```

Reply:
Wouldn't that be redundant as it would be explained in the error anyways? `{e:#}`

Reply:
The error in `{e:#}` is usually not very well formated.

Reply:
0052b019739bcd87a3455203c16bf83246e276f6